### PR TITLE
Add Theia

### PR DESCRIPTION
**Pull request by svenefftinge**

### CODE REVIEW

**Comment by styfle** [comment]
@svenefftinge Thanks for the PR!

A couple things:

Can you change the description to follow the [contributing.md](https://github.com/styfle/awesome-online-ide/blob/master/contributing.md) guidelines?

Is this hosted online somewhere so I can try it?

**Comment by svenefftinge** [comment]
> Is this hosted online somewhere so I can try it?

There is https://gitpod.io which currently is in private beta. I will be happy to give you access.
Otherwise you could try the docker images.

File: README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ A curated list of awesome online implementations of integrated development envir
...
 * [Wide](https://wide.b3log.org/) [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/b3log/wide) <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Run Go with full intellisense code-completion, expression-info, and jump to declaration
 * [Glitch](https://glitch.com/) - Run full Node.js projects with support for automating hosting, version control, CDN for assets, secure storage of secrets, real-time collaboration, full console access, GitHub import/export, server-side debugger, instant deployments, and more
+* [Theia](https://theia-ide.org/) - [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/theia-ide/theia) [<img title="Self Hosted" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/docker.svg" />](https://hub.docker.com/u/theiaide/) - Run any language on a full VM complete with a terminal built on Vs Code's [monaco editor](https://github.com/Microsoft/monaco-editor).
```
Comment (styfle) [inline README.md]: Change "built on Vs Code's" to "built on".

Comment (styfle) [reply README.md]: Change the link to monaco to https://microsoft.github.io/monaco-editor/

Comment (styfle) [reply README.md]: Add another github logo that links to https://github.com/Microsoft/monaco-editor

Comment (styfle) [reply README.md]: Remove the period a the end &mdash; [this is not a complete sentence](https://github.com/styfle/awesome-online-ide/blob/master/contributing.md)

**Comment by styfle** [comment]
Is the relationship between Gitpod.io and Theia, similar to the relationship between CodeEnvy.io and Eclipse Che?

Maybe it would make more sense to list them on the same line like we do for the CodeEnvy/EclipseChe?

**Comment by svenefftinge** [comment]
No, Theia is a diverse open-source project hosted at Eclipse. 
While TypeFox (company behind Gitpod) started it together with Ericsson. The contributor list now includes many other individuals and companies (e.g. RedHat, IBM, Arm).
Actually, the Eclipse Che team is moving to replacing their GWT based IDE with Theia, too.

I will make an additional PR for Gitpod, once it has left private beta.

**Comment by styfle** [comment]
Thanks :+1:
Once you fix the comments, I can merge it

**Comment by svenefftinge** [comment]
Done. Please let me know if I missed something.

**Comment by styfle** [comment]
Excellent, thanks 👍🏼

**Pull request merged**